metadata-io/src/main/java/com/linkedin/metadata/search/transformer/SearchDocumentTransformer.java

```diff
@@ -30,40 +31,39 @@
   // The cap improves search speed when having fields with a large number of elements
   private final int maxArrayLength;
 
-  public Optional<String> transformSnapshot(
-      final RecordTemplate snapshot,
-      final EntitySpec entitySpec,
-      final Boolean forDelete
-  ) {
-    final Map<SearchableFieldSpec, List<Object>> extractedFields =
+  public Optional<String> transformSnapshot(final RecordTemplate snapshot, final EntitySpec entitySpec,
```

Comment:
Question - does this function really accept snapshots?

Reply:
Yeah legacy. It's why we have two functions here. one for snapshot based entities and one for not. Not sure where it is still used tho.